types/gloas/execution_payload_bid.yaml

```
@@ -0,0 +1,36 @@
+Gloas:
+  ExecutionPayloadBid:
+    type: object
+    description: "The [`ExecutionPayloadBid`](https://github.com/ethereum/consensus-specs/blob/00d531949b1f30516979b60ddd2a411e7f388299/specs/gloas/beacon-chain.md#executionpayloadbid) object from the CL Gloas spec."
+    required: [parent_block_hash, parent_block_root, block_hash, fee_recipient, gas_limit, builder_index, slot, value, blob_kzg_commitments_root]
```

`execution_payment` missing

```suggestion
    required: [parent_block_hash, parent_block_root, block_hash, fee_recipient, gas_limit, builder_index, slot, value, execution_payment, blob_kzg_commitments_root]
```